force-app/main/default/triggers/Count.trigger

```diff
@@ -0,0 +1,70 @@
+trigger Count on Task (after insert, after update, after delete) {
+  
+    
+      Set<Id> accIds = new Set<Id>();
+        if (trigger.isInsert) {
+            for (Task task : Trigger.new){
+                if (task.whatId !=null && String.valueOf(task.whatId).startsWith('001')) {
+                    accIds.add(task.whatId);
+                }
+            }
+        } else if (trigger.isUpdate) {
+            for (Task task : Trigger.new){
+                if (task.whatId !=null && String.valueOf(task.whatId).startsWith('001')) {
+                    accIds.add(task.whatId);
+                }
+            }
+        } else if (trigger.isDelete) {
+            for (Task task : Trigger.old){
+                if (task.whatId !=null && String.valueOf(task.whatId).startsWith('001')) {
+                    accIds.add(task.whatId);
+                }
+            }
+        }
+        
+    if (accIds.size() > 0){
```

Comment:
why is this check important?

Reply:
The 'accIds' Set consists of Ids of Tasks that have whatId not null, so tasks that are related to an account. The trigger fires after any task is created or updated or deleted, but I want to take into consideration only tasks related to accounts. When my Set has 0 elements, it means that the tasks that fired the trigger aren't related to accounts, so the trigger don't need to proceed.

Do you think I don't need this check?